app/main.py

```diff
@@ -1,54 +1,64 @@
 import json
-import xml.etree.ElementTree as ET
+import xml.etree.ElementTree as ElementTree
 
 
 class Book:
-    def __init__(self, title: str, content: str):
+    def __init__(self, title: str, content: str) -> None:
         self.title = title
         self.content = content
 
-    def display(self, display_type: str) -> None:
-        if display_type == "console":
-            print(self.content)
-        elif display_type == "reverse":
-            print(self.content[::-1])
-        else:
-            raise ValueError(f"Unknown display type: {display_type}")
-
-    def print_book(self, print_type: str) -> None:
-        if print_type == "console":
-            print(f"Printing the book: {self.title}...")
-            print(self.content)
-        elif print_type == "reverse":
-            print(f"Printing the book in reverse: {self.title}...")
-            print(self.content[::-1])
-        else:
-            raise ValueError(f"Unknown print type: {print_type}")
+    def display_console(self) -> None:
+        print(self.content)
+
+    def display_reverse(self) -> None:
+        print(self.content[::-1])
+
+    def print_book_console(self) -> None:
+        print(f"Printing the book: {self.title}...")
+        print(self.content)
+
+    def print_book_reverse(self) -> None:
+        print(f"Printing the book in reverse: {self.title}...")
+        print(self.content[::-1])
 
     def serialize(self, serialize_type: str) -> str:
         if serialize_type == "json":
-            return json.dumps({"title": self.title, "content": self.content})
+            return self.serialize_json()
         elif serialize_type == "xml":
-            root = ET.Element("book")
-            title = ET.SubElement(root, "title")
-            title.text = self.title
-            content = ET.SubElement(root, "content")
-            content.text = self.content
-            return ET.tostring(root, encoding="unicode")
-        else:
-            raise ValueError(f"Unknown serialize type: {serialize_type}")
+            return self.serialize_xml()
+
+        raise ValueError(f"Unknown serialize type: {serialize_type}")
+
+    def serialize_json(self) -> json:
+        return json.dumps({"title": self.title, "content": self.content})
+
+    def serialize_xml(self) -> json:
+        root = ElementTree.Element("book")
+        title = ElementTree.SubElement(root, "title")
+        title.text = self.title
+        content = ElementTree.SubElement(root, "content")
+        content.text = self.content
+        return ElementTree.tostring(root, encoding="unicode")
```

Comment:
Your code violates Single-responsibility principle. These methods should be in difference classes